src/pathfinder.js

```diff
@@ -81,8 +82,21 @@ class Pathfinder {
     return cheapestPath
   }
 
+  // params -
+  //   sourceLedger
+  //   destinationLedger
+  //   destinationAmount
+  //   destinationAccount (optional)
```

Comment:
I'm confused: destinationAccount is not in the function signature? Also, what is this comment for?